### PR TITLE
Added script for producing Trello statistics

### PR DESCRIPTION
**Pull request by sabre1041**
Script for producing Trello statistics

Get API key at link provided in readme. Fill in variables in script. Execute

### CODE REVIEW

File: trello-stats.py [hunk 1]
```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env python
```
Comment (etsauer) [inline trello-stats.py:1]: script should be executable.

File: trello-stats.py [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+            member_cards.append(card_id)
+
+            members_cards[member_id] = member_cards
+
+print "=== Statistics for Trello Team '{0}' ====\n".format(org_response['displayName'] if 'displayName' in org_response else org_response['name'])
+for key, value in members_cards.iteritems():
+        member = get_member(session, key)
+        print "{0} has {1} cards".format(member['username'], len(value))
+        for card in value:
+            print "   - Board: {0} | Card: {1}".format(cards[card]['board']['name'], cards[card]['name'])
```
Comment (etsauer) [inline trello-stats.py]: When running the script, I get a few results, but then it fails with the following:
```Traceback (most recent call last):
  File "./trello-stats.py", line 79, in <module>
    print "   - Board: {0} | Card: {1}".format(cards[card]['board']['name'], cards[card]['name'])
UnicodeEncodeError: 'ascii' codec can't encode character u'\u201c' in position 28: ordinal not in range(128)
```

File: README.md [hunk 3]
```diff
@@ -40,6 +40,19 @@ is:closed is:issue archived:false user:redhat-cop assignee:etsauer updated:>=201
...
+
+First, get an API key [here](https://trello.com/app-key)
+
+Then, configure the following variables:
+
+```
+TRELLO_API_KEY = '<API_KEY>'
+TRELLO_API_TOKEN = '<API_TOKEN>'
+```
+
```
Comment (etsauer) [inline README.md:55]: Can we include a sample run of the script with some dummy output?

**Review by etsauer — CHANGES_REQUESTED** [review]
@sabre1041 a few things.

**Comment by sabre1041** [comment]
@etsauer addressed comments

**Comment by sabre1041** [comment]
@etsauer bump

**Review by etsauer — APPROVED** [review]

**Pull request merged**